docs/hub-overview/README.md

```diff
@@ -1,9 +1,9 @@
-<!--
+---
```

Comment:
think these are wrong, should be `<!--`

Reply:
Both work, current vuepress docs recommend YAML as default for frontmatter: https://vuepress.vuejs.org/guide/frontmatter.html